etc/profile-m-z/newsboat.profile

```diff
@@ -16,7 +21,12 @@ include disable-passwdmgr.inc
 include disable-programs.inc
 include disable-xdg.inc
 
-mkdir ${HOME}/.newsboat
+mkdir ${HOME}/.local/share/newsboat
+whitelist ${HOME}/.config/newsbeuter
+whitelist ${HOME}/.config/newsboat
```

Comment:
Requires a `mkdir`/`mkfile` too.

Reply:
it can't start without a configuration, but i forgot importing.